runners/google-cloud-dataflow-java/worker/src/main/java/org/apache/beam/runners/dataflow/worker/StreamingDataflowWorker.java

```diff
@@ -668,6 +669,10 @@ public static void main(String[] args) throws Exception {
       enableBigQueryMetrics();
     }
 
+    if (DataflowRunner.hasExperiment(options, "enable_kafka_metrics")) {
+      KafkaSinkMetrics.setSupportKafkaMetrics(true);
```

Comment:
Instead of setting IO flags in worker, one can utilize the JvmInitializer.runBeforeProcessing mechanism that is introduce a JvmInitializer implementation in org.apache.beam.sdk.io.kafka to initialize the flags on worker. This avoids the need of kafka dependency